src/content/api/loaders.md

```diff
@@ -471,3 +509,96 @@ Hacky access to the Compiler object of webpack.
 ### `this._module`
 
 Hacky access to the Module object being loaded.
+
+
+## Error Reporting
+
+There are some ways to throw out errors from inside a loader:
+
+### Using `this.emitError`
+
+It just report errors but not interrupt module compiling.
```

Comment:
`[this.emitError](/api/loaders/#this-emiterror) will report the errors without interrupting module's compilation.` or any other way you can rephrase this to sound closer to software documentation page